### PR TITLE
Design of Event page

### PR DESCRIPTION
**Pull request by zordican**
ADD: Event page design and Made it responsive

### CODE REVIEW

**Comment by bj-jiwrajka** [comment]
> ADD: Event page design and Made it responsive

Design:

image

Yours:

image

fix this @zordican

**Comment by bj-jiwrajka** [comment]
> ADD: Event page design and Made it responsive

Fix this for smaller devices:

image

File: public/Font/PixeloidSans-Bold.ttf
Comment (byt3h3ad) [inline public/Font/PixeloidSans-Bold.ttf:1]: already present in dev

File: public/Font/PixeloidSans.ttf
Comment (byt3h3ad) [inline public/Font/PixeloidSans.ttf:1]: already present in dev

File: src/Pages/Home/EventPage/EventPage.module.scss [hunk 1]
```diff
@@ -0,0 +1,167 @@
+@import url('https://fonts.googleapis.com/css2?family=JetBrains+Mono&display=swap');
+@font-face{
+    font-family: "Pixeloid";
+    src: local("PixeloidSans"),
+    url("../../../../public/Font/PixeloidSans.ttf") format("truetype");
```
Comment (byt3h3ad) [inline src/Pages/Home/EventPage/EventPage.module.scss]: change to `/fonts/[path].` files in public directory are served at the root path.

File: src/Pages/Home/EventPage/EventPage.module.scss [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+}
+@font-face{
+    font-family: "PixeloidBold";
+    src: local("PixeloidSans-Bold"),
+    url("../../../../public/Font/PixeloidSans-Bold.ttf") format("truetype");
```
Comment (byt3h3ad) [inline src/Pages/Home/EventPage/EventPage.module.scss]: change to `/fonts/[path]`. files in public directory are served at the root path.

**Review by byt3h3ad — CHANGES_REQUESTED** [review]

File: src/App.jsx [hunk 3]
```diff
@@ -1,15 +1,10 @@
-import EventPage from "./Pages/Home/EventPage/EventPage";
-import Home from "./Pages/Home/Home";
-import { BrowserRouter, Routes, Route } from "react-router-dom";
-function App() {
-	return (
-		<BrowserRouter>
-			<Routes>
-				<Route path="/" element={<Home />} />
-				<Route exact path="/Event" element={<EventPage />} />
```
Comment (bj-jiwrajka) [inline src/App.jsx]: add the route to event page

**Review by bj-jiwrajka — CHANGES_REQUESTED** [review]

File: src/Pages/Home/EventPage/EventPage.module.scss [hunk 4]
```diff
@@ -0,0 +1,199 @@
...
+            background-clip: text;  
+            
+        }
+        p{
+            font-family: 'JetBrains Mono', monospace;
+            font-weight: 400;
+            font-size: 1.2rem;
+            line-height: 1.875rem;
+            text-align: justify;
+            color: #C7FFF2;
```
Comment (bj-jiwrajka) [inline src/Pages/Home/EventPage/EventPage.module.scss]: add the color in ```index.scss``` file

**Review by bj-jiwrajka — CHANGES_REQUESTED** [review]

**Comment by bj-jiwrajka** [comment]
@zordican move the ```EventPage``` folder outside the ```Home``` page

File: src/App.jsx [hunk 5]
```diff
@@ -1,10 +1,19 @@
...
 import Home from "./Home";
 import Preview from "./components/Preview/Preview";
+import { Routes, Route } from "react-router-dom";
+import EventPage from "./Pages/EventPage/EventPage";
 
 const App = () => {
 	const [loaded, setLoaded] = useState(false);
-	return loaded ? <Home /> : <Preview setLoaded={setLoaded} />;
+	return (
+		<>
```
Comment (bj-jiwrajka) [inline src/App.jsx]: your Event Page is displaying the content of HomePage too, fix this

**Review by bj-jiwrajka — CHANGES_REQUESTED** [review]

File: src/Pages/EventPage/EventPage.module.scss [hunk 6]
```diff
@@ -0,0 +1,199 @@
...
+    src: local("PixeloidSans-Bold"),
+    url("/fonts/PixeloidSans-Bold.ttf") format("truetype");
+}
+
+.MainDiv{
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 100%;
+    height: 100vh;
```
Comment (bj-jiwrajka) [inline src/Pages/EventPage/EventPage.module.scss]: why is it fixed to 100vh? (_if needed, use min-height)_

File: src/Pages/EventPage/EventPage.module.scss [hunk 7]
```diff
@@ -0,0 +1,199 @@
...
+    background-color: black;
+    flex-wrap: wrap;
+    overflow: hidden;
+    .event_info{
+        width: 50%;
+        height: 100vh;
```
Comment (bj-jiwrajka) [inline src/Pages/EventPage/EventPage.module.scss]: same again :)

due to this, the buttons at the bottom are not visible

**Review by bj-jiwrajka — CHANGES_REQUESTED** [review]

**Comment by byt3h3ad** [comment]
merging.

**Pull request merged**